phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -210,6 +227,34 @@ private void initNetworkPane() {
 
     }
 
+    private void initAboutPane() {
+        aboutPanel = new VBox();
+        aboutPanel.getStyleClass().add(CONFIGURATION_PANE_CSS_CLASS);
+
+        final Text title = new TextWithStyle(translate("About"), TITLE_CSS_CLASS);
+        aboutPanel.getChildren().add(title);
+
+        final GridPane gridPane = new GridPane();
+        gridPane.getStyleClass().add("grid");
+
+        gridPane.add(new TextWithStyle(translate("Name:"), CAPTION_TITLE_CSS_CLASS), 0, 0);
+        gridPane.add(new Label(applicationName), 1, 0);
+
+        gridPane.add(new TextWithStyle(translate("Version:"), CAPTION_TITLE_CSS_CLASS), 0, 1);
+        gridPane.add(new Label(applicationVersion), 1, 1);
+
+        gridPane.add(new TextWithStyle(translate("Git Revision:"), CAPTION_TITLE_CSS_CLASS), 0, 2);
```

Comment:
@plata I think your answer is right. The release versions will be built with a certain revision. Because of this I think it's alright to show the git revision. In addition the version string tells the user that he/she is using a release version.